Lab2/src/test/java/ua/lviv/IoT/lab2/manager/impl/CheckWordTest.java

```diff
@@ -0,0 +1,25 @@
+package ua.lviv.IoT.lab2.manager.impl;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please do not import *. Your Checkstyle should've warned you about this issue.
[Why not import all?](https://www.reddit.com/r/java/comments/2i4uc7/comment/ckyu8v4/?utm_source=share&utm_medium=web2x&context=3)

Reply:
Also fixed it.